### ISSUE TITLE
Add GCC 9

### ISSUE THREAD

**Issue by Lyberta** [conan-io/conan#5093]
GCC 9 was released today and is working fine when manually added to `settings.yml`.

You can add it to default one.

**Comment by memsharded** [comment]
It has been already added in https://github.com/conan-io/conan/pull/5046 will be released in Conan 1.15 very soon. Thanks for reporting!

**Comment by v-strob** [comment]
I see having an option to disable version checking has already been rejected before:
#2650
Any possibility of revisiting this?

I like testing my active projects with the latest compilers shortly after they come out so I can get ready to upgrade. By doing so with Conan, I'll likely have to maintain a custom settings.yml. It isn't a one-time thing either, I have to keep updating it with other valuable settings that get added in future conan releases.

If you must keep the check, you could web scrape the compiler sites and get notified when betas are posted so you can set the version in advance of the actual release.

**Comment by memsharded** [comment]
Hi @v-strob 

While I still understand that request, there are a few things to consider:
- Often, specially for major versions, it is not enough to add them to settings.yml. For full functionality, a couple of things more have to be added to conan, like defining what is the default ``cppstd`` they implement. 
- Any weird behavior that comes out from unmodeled things of the new compilers, will produce new support requests, filing issues, etc., and responding to all support requests is becoming a challenge.
- What if we allow relaxed settings, you create packages with those relaxed settings (lets say compiler.version=11.3 for a new gcc version), and the moment we really standardize those settings, they result in a different package-id and all the packages you created are no longer valid? I see a bunch of complains about broken behavior, but this is a likely scenario.
- Web scraping is a new development and maintenance effort that we cannot afford now either. We try to do our best processing issues and contributions regarding new compilers from interested users.

In short: the flexibility and "relaxed" Conan behaviors come with a high price, with later big challenges to keep our commitment to stability and "non breaking", and an added effort in support and maintenance. While we do our best to try to provide it whenever possible, for this one, it is possible to have a reasonable workaround, and even if not a one-time thing, is a thing that has to be done only like once every few months (not that a new compiler is out every day), and it really takes just a couple of minutes. So for the sanity of the maintainers, I'd say it is better not to provide this at this moment. However, it is something that we have in mind, don't think that we don't know it, and we will keep it in mind for the future. Thanks for the feedback!

**Comment by v-strob** [comment]
Ok, thanks for the context into the challenges and risks of relaxing settings.  Also, thanks for the gcc 9.1 update, I updated my packages last night.

With regard to web-scraping, here is a Python script I use:
https://gitlab.com/ssrobins/tools

It's set to run once a day and a failure triggers an email listing what's been upgraded:
https://gitlab.com/ssrobins/tools/-/jobs/208093619

If it's helpful, feel free to use all or part of it.
